### DISCUSSION TITLE
Newbie Question - Remote Access?

### DISCUSSION THREAD

**Discussion by alliparker19** [stashapp/stash#2243] [Q&A] [answered]
I am currently using Plex but thought I would give this a try. So far everything looks great locally but how do I go about getting remote access? Please advise, thank you!

**Comment by stg-annon** [comment, marked as answer]
So stash does not support a turnkey solution for remote access you would need to provide that yourself, the most straight forward solution would be to port forward the port that stash is using (default: `9999`) to the computer running stash and set a password for stash, another more preferred/secure solution is to VPN back into the same network as stash, a popular VPN provider for this use case is Tailscale

**Reply by crazycuck** [reply]
Two cents:

Also cloudflared based solution such Zero Trust where you can enforce device, user, mfa based requirements to reach those services.

**Reply by feederbox826** [reply]
Cloudflare throttles non-html connections if proxied
